classes/class-twentytwenty-customize.php

```diff
@@ -252,13 +251,13 @@ public static function register( $wp_customize ) {
 				array(
 					'title'       => __( 'Cover Template', 'twentytwenty' ),
 					'capability'  => 'edit_theme_options',
-					'description' => __( 'Settings for the "Cover Template" page template.', 'twentytwenty' ),
+					'description' => __( 'Settings for the "Cover Template" page template. Add a featured image to use as background.', 'twentytwenty' ),
 					'priority'    => 42,
 				)
 			);
 
 			/* Overlay Fixed Background ------ */
-
+			
```

Comment:
I believe this space will trigger en Travis error, @carolinan.

Reply:
I just see that you solved it in https://github.com/WordPress/twentytwenty/pull/787/commits/9f5c8db319ebdb6cc4fa544ad394191916326d83 already! 😁